### ISSUE TITLE
[BUG]: Attaque puissante fantassin, vitesse -15% pas bien assignée

### ISSUE THREAD

**Issue by Evatco** [DraftBot-A-Discord-Adventure/DraftBot#2404]
### Description du bug.

Le -15% de vitesse de l'attaque du fantassin est censé se mettre à l'adversaire alors que c'est le fantasin qui se le prends 

### Reproduction du bug.

Faire un combat en fantassin et faire une attaque puissante et constater la baisse de vitesse dans les stats 
Ou alors changer le document dans /help fight x)

### Captures d'écran

![Capture d’écran du 2024-02-12 14-09-12](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/138615431/89d3a36b-fa33-48c8-adf9-4d723380ae62)
![Capture d’écran du 2024-02-12 14-09-21](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/138615431/c6b72602-59dc-4728-95c7-e52202698c58)
![Capture d’écran du 2024-02-12 14-09-47](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/138615431/cb97b3a3-bb4e-43ff-b25f-bc8f44fd4ea1)

### Sur quel shard avez vous rencontré le bug ?

0 mais je pense tous juste celui sur lequel je suis pour le moment (je découvre juste que c'est un bug)

**Comment by KirIcare** [comment]
Déjà signalé y'a un bon moment (pendant la grosse maj des fights et équilibrage des classes). On avait pas ouvert d'issues puisque c'était pendant l'équilibrage, ça devait être fait au passage mais on a jamais eu la réponse de "on change la documentation ou la cible du -15%de vitesse ?"

**Comment by Evatco** [comment]
Ah d'accord je savais pas mais je pense que le ieux ça serait la cible du -15% car ça buff un peu la classe qui n'est pas très appréciée masi après je suis fantassin donc mon point de vue est biaisé

**Comment by KirIcare** [comment]
Non, de un tu as bien fait maintenant y'a une vrai trace écrite et de deux je suis d'accord avec toi, la classe fantassin a pas mal de désavantages, donc si on se débarrasser du -15% de vitesse sur la puissance (qui n'est clairement pas ce qui a de plus handicapant) c'est déjà ça

**Comment by Evatco** [comment]
J'espère juste que ça va pas passer après la v5 vu que me parait pas compliquer à changer mais j'y connais rien xD

**Comment by BastLast** [comment]
Donc ca sera après la v5

**Comment by Evatco** [comment]
Je comprends je comprends

**Comment by BastLast** [comment]
sera fix par #2719
